docling_eval/dataset_builders/xfund_builder.py

```diff
@@ -248,6 +268,21 @@ def populate_key_value_item(
             )
             cell_by_id[cell_id] = cell
 
+            for word in item.get("words", []):
+                text = word.get("text", "")
+                bbox = word.get("box", [0, 0, 0, 0])
```

Comment:
Text without bounding boxes would better be discarded. We should have rather incomplete than false information. Correct bounding boxes are a key part of a SegmentedPage representation.